### ISSUE TITLE
Use native Object.assign instead of xtend

### ISSUE THREAD

**Issue by jimmywarting** [mcollina/autocannon#169]
[xtend](https://github.com/Raynos/xtend) can easily be replaced with [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)

```js
// immutable
Object.assign({}, a, b)

// mutable
Object.assign(a, b)
```

**Comment by mcollina** [comment]
Can you please send a PR?

Il giorno mar 16 ott 2018 alle 16:21 Steven <notifications@github.com> ha
scritto:

**Comment by thekemkid** [comment]
We should do it. When we wrote

**Comment by thekemkid** [comment]
Closed via #173
